packages/simple-hub/package.json

```diff
@@ -34,6 +34,7 @@
     "eslint-plugin-jest": "23.6.0",
     "eslint-plugin-prettier": "3.1.2",
     "firebase-server": "^1.1.0",
+    "ganache-cli": "^6.9.1",
```

Comment:
Maybe ensure it's the same version as other packages to reduce unnecessary `node_modules` bloat

Reply:
Good call. 8e9f64018